passes/techmap/alumacc.cc

```diff
@@ -331,9 +331,6 @@ struct AlumaccWorker
 			if (GetSize(C) > 1)
 				goto next_macc;
 
-			if (!subtract_b && B < A && GetSize(B))
-				std::swap(A, B);
-
```

Comment:
In hindsight, this can remain, as `cmp` unification now is order-independent. This also allows the tests to pass.